### ISSUE TITLE
Training loop

### ISSUE THREAD

**Issue by carrascomj** [carrascomj/gcn-prot#7]
* [x] DataLoader.
* [x] Do the transformations.
* [x] Convert to parameter.
* [x] Some simple visualization of the loss.
